app/views/About.js

```diff
@@ -123,7 +124,7 @@ const styles = StyleSheet.create({
     flexDirection: 'column',
     width: '100%',
     backgroundColor: Colors.INTRO_WHITE_BG,
-    paddingHorizontal: 26,
+    paddingHorizontal: 40,
```

Comment:
its not clear to me how this keeps the page from scrolling?  If the intent is to remove the scrolling could we change the `<ScrollView>` to a `<View>`?

Reply:
Hi John, thanks for your feedback. When I tried changing it to a View, a lot of the CSS was distorted, which led to the more “hacky” solution of just increasing the padding slightly so that the scroll view doesn’t scroll anymore. Hope this isn’t too big of an issue.

Reply:
Stylesheet* not CSS

Reply:
Hmm, certainly wouldnt call it a big issue, but ideally we are laying out our views in such a way that we can be declarative without oddities in the look of it.  That said i think any clean up here would make sense in a future commit.

Once you get the formatting validation passing we can merge this in!